lib/opencensus/stats/measurement.rb

```diff
@@ -3,9 +3,33 @@
 
 module OpenCensus
   module Stats
-    # Struct that holds measurement value
-    # @attr [Measure] Measure details
-    # @attr [Integer, Float] Value of the measurement
-    Measurement = Struct.new(:measure, :value)
+    # Measurement
+    #
+    # Describes a data point to be collected for a Measure.
+    class Measurement
+      # @return [Measure] A measure to which the value is applied.
+      attr_reader :measure
+
+      # @return [Integer,Float] The recorded value
+      attr_reader :value
+
+      # @return [TagMap] The tags to which the value is applied
+      attr_reader :tags
+
+      # @return [Time] The time when measurement was created.
+      attr_reader :time
+
+      # Create a instance of measurement
+      #
+      # @param [Measure] measure A measure to which the value is applied.
+      # @param [Integer,Float] value Measurement value.
+      # @param [Hash<String,String>] tags The tags to which the value is applied
+      def initialize measure:, value:, tags:
+        @measure = measure
+        @value = value
+        @tags = Tags::TagMap.new tags
+        @time = Time.now
```

Comment:
Any time you create `Time.now` timestamps, can you please convert to utc?

Reply:
Updated all `Time.now` instances with `Time.now.utc`